src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -245,37 +245,27 @@ private void endStream(RowStream<Row> stream, RoutingContext ctx,
   }
 
   Future<UUID> upsertTeEntry(TenantPgPool pool, SqlConnection con, RoutingContext ctx,
-                             String counterReportTitle, String match) {
+                             String counterReportTitle, String printIssn, String onlineIssn) {
```

Comment:
Declaring `final String onlineIssn` avoids the additional `match` variable.

Reply:
Using onlineIssn would suffice. The reason for declaring this variable this way is that I think we want to generalize the match key a bit. I hope that's ok.